src/Catalog.API/Apis/CatalogApi.cs

```diff
@@ -73,15 +58,14 @@ public static IEndpointRouteBuilder MapCatalogApiV1(this IEndpointRouteBuilder a
             .WithTags("Brands");
 
         // Routes for modifying catalog items.
-        api.MapPut("/items", UpdateItem)
-            .WithName("UpdateItem")
-            .WithSummary("Create or replace a catalog item")
-            .WithDescription("Create or replace a catalog item")
-            .WithTags("Items");
         api.MapPost("/items", CreateItem)
             .WithName("CreateItem")
             .WithSummary("Create a catalog item")
             .WithDescription("Create a new item in the catalog");
+        api.MapPut("/items/{id:int}", UpdateItem)
```

Comment:
Do we want to retain the `Items` tag on this and the other items-related endpoints?